src/components/Attachments/AttachmentView/AttachmentViewPdf/index.android.js

```diff
@@ -15,17 +15,27 @@ function AttachmentViewPdf(props) {
     const offsetX = useSharedValue(0);
     const offsetY = useSharedValue(0);
 
+    // Reanimated freezes all objects captured in the closure of a worklet.
+    // Since Reanimated 3, entire objects are captured instead of just the relevant properties.
+    // See https://github.com/software-mansion/react-native-reanimated/pull/4060
+    // Because context contains more properties, all of them (most notably the pager ref) were
+    // frozen, which combined with Reanimated using strict mode since 3.6.0 was resulting in errors.
+    // Without strict mode, it would just silently fail.
+    // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze#description
+    const isContextReady = attachmentCarouselPagerContext !== null;
```

Comment:
If `attachmentCarouselPagerContext` is nullable, app will crash on next line

Reply:
True, I'll change the logic.

Reply:
It will cause regression - #33357

Reply:
What will cause a regression?

Reply:
`const shouldPagerScroll = attachmentCarouselPagerContext.shouldPagerScroll;` without null check
This was the fix: https://github.com/Expensify/App/pull/33383

Reply:
Can you check if it is ok now?

Reply:
This thread can be marked as resolved